includes/Media/Video/Cropped.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Class Cropped
```

Comment:
Let's rename `Cropped` -> `Cropping` for consistency with other classes (`Muting`, `Trimming`, etc.)